### DISCUSSION TITLE
Google Driveに保存した画像を埋め込むと表示されたりされなかったりする

### DISCUSSION THREAD

**Discussion by eizaburo** [increments/qiita-discussions#333] [要望・アイデア・不具合]
お世話になります。
Google Driveに保存した画像を一般公開設定（リンク取得）し、Qiitaに埋め込みをすると、同じ画像でも表示されたり、されなかったりします。画像がうまく表示されてないときに画像リンクをクリックすると「Not Found」となります。しばらく放っておくとまた表示されたりします。Google Drive上の画像を利用するに際し何か注意点などありますでしょうか？

（以下のような現状が発生します）

・Google Drive上で画像公開しURL取得
・Qiitaに埋め込む（編集画面で普通に見える）
・公開（普通に見える）
・しばらくすると見えなくなる（もちろん何もいじってない）
・しばらくすると見えたりする
・あとは、見えたり、見えなかったりの繰り返し

気のせいかもしれませんが朝とか夜とかは普通に見え、昼間おかしくなることが多いように感じます（camoキャッシュサーバ等の負荷？）。
なお、Qiitaチームも利用させていただいておりますが、そちらではそのような現象は今の所発生していません。

よろしくお願いします。

表示できないときは以下のように編集画面でも表示できません。

<img width="1150" alt="image" src="https://user-images.githubusercontent.com/3616214/196604100-6127a888-aa27-4950-83a9-cdb4ebce5349.png">

**Comment by tanaka-asami415** [comment]
@eizaburo 
ご報告ありがとうございます🙏
ご不便をおかけして申し訳ありません。こちら調査いたします🙇

**Comment by kyntk** [comment]
@eizaburo
お待たせしており、申し訳ございません。

詳しく調査をするために追加で情報をいただきたいです。
大変申し訳ないのですが、次に画像が表示されていないのを発見したときに、その日時と、表示されなかった画像のURLを教えていただけますでしょうか。 :bow:

**Reply by kyntk** [reply]
@eizaburo その後どうでしょうか？
画像が表示されない事象は発生していないでしょうか？
